app/src/main/java/com/arctouch/codechallenge/data/MoviePagingListData.java

```diff
@@ -0,0 +1,43 @@
+package com.arctouch.codechallenge.data;
+
+import android.annotation.SuppressLint;
+import android.util.Log;
+import androidx.annotation.NonNull;
+import com.arctouch.codechallenge.data.model.Movie;
+
+public class MoviePagingListData extends MovieBasePagedKeyedDataSource {
+
+    private MoviesDataSource dataSource;
+    private static final int FIRST_PAGE = 1;
+
+
+    public MoviePagingListData (MoviesDataSource dataSource) {
+        this.dataSource = dataSource;
+    }
+
+    @SuppressLint("CheckResult")
```

Comment:
Why is the warning happening? Is suppressing it the best option or could the issue actually impact your code?

Reply:
This Warning is happening because Lint thinks that the results of my Observable is being used. But the truth is, the results are being used. It is something that I forgot to do in this app, it was to put these observables in a Disposable bag, it would correct the warning.